### PR TITLE
[Github-Pages.xml] added exclusions from #4151 and fixed #4372

### LINKED ISSUE #4372: GitHub Pages rule breaks site
http://ibotpeaches.github.io/Apktool/

Site will reload repeatedly when the GitHub Pages rule is active.

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: src/chrome/content/rules/Github-Pages.xml [hunk 1]
```diff
@@ -5,6 +5,28 @@
...
+	<exclusion pattern="^http://ibotpeaches\.github\.io/Apktool/" />
+	<exclusion pattern="^http://schizoduckie\.github\.io/DuckieTorrent/" />
+	<exclusion pattern="^http://schizoduckie\.github\.io/PimpMyuTorrent/" />
+	<exclusion pattern="^http://leaflet\.github\.io/" />
+
+	<test url="http://duckietv.github.io/DuckieTV/" />
+	<test url="http://ibotpeaches.github.io/Apktool/" />
+	<test url="http://schizoduckie.github.io/DuckieTorrent/" />
+	<test url="http://schizoduckie.github.io/PimpMyuTorrent/" />
+	<test url="http://leaflet.github.io/Leaflet.markercluster/example/marker-clustering-realworld.388.html" />
```
Comment (danzel) [inline src/chrome/content/rules/Github-Pages.xml]: The leaflet.github.io rule can be removed now, it has been corrected to work.

Comment (J0WI) [inline src/chrome/content/rules/Github-Pages.xml]: When greping though the repo I can still find some hardcoded `http:` links, but the initial issue seems to be fixed.

**Comment by J0WI** [comment]
ping @fuglede for review

File: src/chrome/content/rules/Github-Pages.xml [hunk 2]
```diff
@@ -5,6 +5,26 @@
 
 	<target host="*.github.io" />
 
+
+	<!-- pages that require http -->
+	<exclusion pattern="^http://duckietv\.github\.io/DuckieTV/" />
```
Comment (fuglede) [inline src/chrome/content/rules/Github-Pages.xml]: A couple of issues here:

* http://duckietv.github.io/DuckieTV (without the trailing slash) still redirects to the HTTPS version.
* I'm actually seeing some CORS issues on http://duckietv.github.io/DuckieTV/ that may or may not be related:

skaermbillede fra 2016-06-16 22 47 28

**Comment by J0WI** [comment]
Added exclusion also without trailing slash.
I think the CORS error is not related, I can't see any difference with HTTPS Everywhere disabled.

**Comment by fuglede** [comment]
Ruleset LGTM, but what is the idea behind taking the exceptions from #4151; that that one had unrelated changes at the time of the PR?

**Comment by J0WI** [comment]
I didn't expect an update on #4151 in near time and picked the bug fixes here.
It doesn't matter which one gets merged as long as they are still about the same.

**Pull request merged**